### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by scott-ace-newton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,167 @@
...
+```
+
+
+Set all the required variables
+------------------------------
+
+```bash
+## Get a new etcd token for a new cluster, 5 refers to the number of initial boxes in the cluster:
+## `curl https://discovery.etcd.io/new?size=5`
+export TOKEN_URL=`curl https://discovery.etcd.io/new?size=5`
```
Comment (tamas-molnar) [inline README.md]: Size should be 3

Comment (BerniVarga) [inline README.md]: We could change the size to 3, cause we only have 3 machines in the publishing cluster.

Comment (scott-ace-newton) [inline README.md]: Thanks

**Comment by kovacshuni** [comment]
+1

**Comment by BerniVarga** [comment]
+1

**Pull request merged**